src/lib/Sage/PostCreateProject.php

```diff
@@ -35,19 +35,39 @@ public static function updateHeaders(Event $event)
         }
     }
 
-    public static function removeBootstrap(Event $event)
+    public static function selectFramework(Event $event)
     {
         $io = $event->getIO();
 
         if ($io->isInteractive()) {
-            if ($io->askConfirmation('<info>Remove Bootstrap?</info> [<comment>y,N</comment>]? ', false)) {
-                file_put_contents('package.json', str_replace('    "bootstrap": "^4.0.0-alpha.6",' . "\n", '', file_get_contents('package.json')));
-                file_put_contents('assets/styles/main.scss', str_replace('@import "~bootstrap/scss/bootstrap";' . "\n", '', file_get_contents('assets/styles/main.scss')));
-                file_put_contents('assets/scripts/main.js', str_replace('import \'bootstrap/dist/js/bootstrap\';' . "\n", '', file_get_contents('assets/scripts/main.js')));
-                file_put_contents('assets/styles/components/_comments.scss', '');
-                file_put_contents('assets/styles/components/_forms.scss', '');
-                file_put_contents('assets/styles/components/_wp-classes.scss', '');
-                file_put_contents('assets/styles/layouts/_header.scss', '');
+            $frameworks = [
+                'Bootstrap',
+                'Foundation',
+                'None'
+            ];
+            $framework = $io->select('<info>Select a CSS framework</info> <comment>(Default: Bootstrap)</comment>', $frameworks, 0);
+
+            switch($framework) {
+                case 0:
+                    break;
+                case 1:
+                    file_put_contents('package.json', str_replace('    "bootstrap": "^4.0.0-alpha.6",' . "\n", '    "foundation-sites": "6.3.0",' . "\n", file_get_contents('package.json')));
```

Comment:
We should turn this into a `preg_replace` to so we aren't specific versions in here as much as possible.

Something like:

```php
$default_framework_pattern = '/"bootstrap": ".*"/';
file_put_contents('package.json', preg_replace($default_framework_pattern, '"foundation-sites": "6.3.0"', file_get_contents('package.json')));
```

Also notice that this eliminates specifying whitespace